webapp/dojo_news/src/App.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <div>
+    <h1>News</h1>
+    <div
+        v-for="item in sortedNewsListItems"
+        :key="item.title">
+      <NewsListItem
+          :title="item.title"
+          :votes="item.votes"
+          @upvote="item.votes++"
+          @downvote="item.votes--"
+          @remove="newsListItems.splice(newsListItems.indexOf(item), 1)"
```

Comment:
```suggestion
          @remove="remove"
      // ...
      methods: {
        remove(removedItem){
          this.newsListItems = this.newsListItems.filter(item => item !== removedItem)
        }
      }
```
...is much better to read than `.splice` which messes around with indices.

Reply:
Thanks!
We tried this in some way and had issues with `computed` clearing its cache.
Also, while performance is not an issue at all here, `filter` checks through the whole array and duplicates it, whereas `splice` only checks the whole array in the worst case and acts in-place.

If we get it to work, we'll probably switch to using `filter` to improve legibility.